src/app/common/linshare-api.service.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+angular.module('linagora.esn.linshare')
+  .factory('linshareApiService', linshareApiService);
```

Comment:
This will cause an issue in Webpack production mode. It should be `linshareApiServiceFactory`